ICS372_Dodd_gp1/src/Driver.java

```diff
@@ -0,0 +1,95 @@
+import java.util.Scanner;
+
+/**
+ * Driver class for running application. Contains main function.
+ */
+public class Driver {
+
+	public static void main(String[] args) {
+		// TODO Write Code
+		System.out.println("Running...");
+		System.out.println("Done.");
+		showUserInterface();
+
+		return;
+	}
+
+	public static void showUserInterface() {
+
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Welcome to small co-op blah blah"); //fix introduction
+		String menu = "Make a selection, enter: \n"
+				+ "1) Enroll a member\n2) Remove a member\n3) Add a product\n"
+				+ "4) Check out a member's items\n5) Process a shipment\n"
+				+ "6) Change the price of a product\n7) Retrieve product info"
+				+ "\n8) Retrieve member info\n9) Print transactions\n"
+				+ "10) List all outstanding orders\n11) List all members with member info\n"
+				+ "12) List all products with product info\n13) Save\n14) Help \n"
+				+ "Press 0 at any time to quit the application";
+		System.out.println(menu);
+		boolean continueApplication = true;
+		while( continueApplication ) {
+			System.out.print("Enter: ");
+			int userInput;
+			try {
+				userInput = scanner.nextInt();
+				switch( userInput ) {
```

Comment:
According to the professor's coding standards, there should be no spaces after left parentheses or before right parentheses.